scripts/test

```diff
@@ -1,23 +1,63 @@
 #!/bin/bash
-echo 'Waiting for ES to start'
-until curl --output /dev/null --silent --head --fail http://es:9200; do
-    printf '.'
-    sleep 5
-done
-echo "ES has started"
-
-exit_status=0
-
-echo  "Test max file descriptors"
-expected_max=1048576
-max_file_descriptors=$(curl http://es:9200/_nodes/stats/process?filter_path=**.max_file_descriptors | jq '.. | .max_file_descriptors? | select(type != "null")')
-
-if [ "$max_file_descriptors" -eq $expected_max ]
-then
-  echo "Max file descriptors test passed. Expected $expected_max and got $max_file_descriptors"
-else
-  exit_status=1
-  echo "Max file descriptors test failed. Expected $expected_max but got $max_file_descriptors"
-fi
+get_es_stats() {
+  echo 'Waiting for ES to start'
+  until curl --output /dev/null --silent --head --fail http://es:9200; do
+      printf '.'
+      sleep 5
+  done
+  echo "ES has started"
+
+  echo "Waiting for up to 60 seconds for the cluster status be at least yellow"
+  curl --silent --output /dev/null "http://es:9200/_cluster/health?wait_for_status=yellow&timeout=60s"
```

Comment:
I'd argue that the health output here has a use in diagnosing issues when ES does not start.

Tried this with a short timeout on ES start up (using `&pretty`) and got this. 

```
$ curl "http://elasticsearch:9200/_cluster/health?wait_for_status=yellow&timeout=6s&pretty"
{
  "cluster_name" : "profiles-db-elastic",
  "status" : "red",
  "timed_out" : true,
  "number_of_nodes" : 1,
  "number_of_data_nodes" : 1,
  "active_primary_shards" : 0,
  "active_shards" : 0,
  "relocating_shards" : 0,
  "initializing_shards" : 4,
  "unassigned_shards" : 6,
  "delayed_unassigned_shards" : 0,
  "number_of_pending_tasks" : 0,
  "number_of_in_flight_fetch" : 0,
  "task_max_waiting_in_queue_millis" : 0,
  "active_shards_percent_as_number" : 0.0
}
```
Maybe a check on `timed_out` before proceeding to the stats command would be useful?

Also, got the following error messages while ES was just starting which are not written to console when using the --silent flag

```
$ curl --fail "http://elasticsearch:9200/_cluster/health?wait_for_status=yellow&timeout=6s&pretty"
curl: (7) Failed to connect to elasticsearch port 9200: Connection refused
$ echo $?
7
$ curl --fail "http://elasticsearch:9200/_cluster/health?wait_for_status=yellow&timeout=6s&pretty"
curl: (52) Empty reply from server
$ echo $?
52
```

so my suggestion to add --silent was incorrect. Sorry!

The tests on document count will obviously fail later but this is about failing early.